request.js

```diff
@@ -79,6 +80,7 @@ inherits(TChannelRequest, EventEmitter);
 
 TChannelRequest.defaultRetryLimit = 5;
 TChannelRequest.defaultTimeout = 100;
+TChannelRequest.defaultMaxRetryRatio = 1.0;
```

Comment:
the default should be `null` ? That's backwards compatibility, alternatively the default can be `5.0` to match the `defaultRetryLimit`